cmake/Goturn.cmake

```diff
@@ -4,8 +4,8 @@ log("Looking for GOTURN files under ${CMAKE_CURRENT_SOURCE_DIR}")
 if (EXISTS ${GOTURN_PROTOTXT} AND EXISTS ${GOTURN_CAFFEMODEL})
     log("GOTURN files found")
     set(GOTURN_FILES_FOUND TRUE)
-    configure_file(${GOTURN_PROTOTXT} goturn.prototxt COPYONLY)
-    configure_file(${GOTURN_CAFFEMODEL} goturn.caffemodel COPYONLY)
+    #configure_file(${GOTURN_PROTOTXT} goturn.prototxt COPYONLY)
```

Comment:
This is causing crash issues on CLion but works okay doing it manually with cmake, disabling for now